worksheet.md

```diff
@@ -181,102 +181,90 @@ time.sleep(2)
 tone_obj.stop()
 ```
 
-Either copy and paste or enter the code below. Pay attention to the `GPIO.setup` command. This line is doing two things: it is setting GPIO pin 4 as an input, **and** setting the internal pull up resistor on it. If you want to use the pull down resistor, you'll need to use `GPIO.PUD_DOWN` instead.
+Either copy and paste or enter the code below. Pay attention `key = gpio.Button(pin, pull_up=True)` line. This assigns the variable `key` to an *instance* of the class `gpio.Button`. The `gpio.Button` class holds all the important information about how to read inputs from the button, and comes with some *methods*, which are like functions, but are attached to the instance of the class, and can access its internal variables.
 
-There is then a `while` loop, which continually reads the state of GPIO pin 4 and prints HIGH or LOW to the screen every second.
+It also tells the class to set GPIO pin 4 as input, then set the internal pull up resistor on it. If you want to use the pull down resistor, you'll need to use `pull_up=False` instead.
+
+Then, there is then a `while` loop, which continually reads the state of GPIO pin 4 and prints HIGH or LOW to the screen every second.
 
 ```python
 pin = 4
-GPIO.setmode(GPIO.BCM)
-GPIO.setup(pin, GPIO.IN, pull_up_down=GPIO.PUD_UP)
+key = gpio.Button(pin, pull_up=True)
 
 while True:
-    reading = GPIO.input(pin)
-    print("HIGH" if reading else "LOW")
+    reading = key.is_pressed
+    print("ON" if reading else "OFF")
```

Comment:
Are `ON` and `OFF` appropriate names for the values? They aren't accurately represented by `HIGH` and `LOW` anymore, as that is handled internally by `gpiozero`, so should they be anything different, i.e `UP` and `DOWN`?